runtime/resolver.go

```diff
@@ -366,6 +424,7 @@ type cachedResolverResult struct {
 
 	// Iterator fields. Should only be populated on short-lived copies obtained via copy().
 	rows []map[string]any
+	meta []map[string]any
```

Comment:
The `meta` field belongs up alongside `data` and `schema`, not here under "Iterator fields"